maphooks/useBreadcrumbs.js

```diff
@@ -1,6 +1,7 @@
 import { useRef, useEffect, useState} from 'react'
+import mapboxgl from '!mapbox-gl'; // eslint-disable-line import/no-webpack-loader-syntax
```

Comment:
Heads up @lowriech, I updated this import to match the usage in useMap. The site wouldn't load without it ("mapbox-gl has not exported members")

Reply:
👍